bitcoin/txn.py

```diff
@@ -52,6 +55,9 @@ def addOutput(self, amount, pkScript):
 		self.outputs.append( (amount, pkScript) )
 	
 	def disassemble(self, retExtra = False):
+		if self.data[4:6] == b'\0\1':
```

Comment:
Why would it? A transaction with the witness dummy/flags should be entirely invalid pre-segwit.